src/stark_verifier/air/stark_proof.cairo

```diff
@@ -5,18 +5,16 @@ from stark_verifier.air.pub_inputs import PublicInputs
 from stark_verifier.air.transitions.frame import EvaluationFrame
 from stark_verifier.utils import Vec
 
-// https://github.com/novifinancial/winterfell/blob/0b7cc3dc28c6b1ad43eb3f2850f644bff1423cf9/air/src/options.rs#L82
 struct ProofOptions {
     num_queries: felt,
     blowup_factor: felt,
     grinding_factor: felt,
-    hash_fn: felt, // https://github.com/novifinancial/winterfell/blob/0b7cc3dc28c6b1ad43eb3f2850f644bff1423cf9/air/src/options.rs#L20
```

Comment:
The idea here was to link to the original Rust code for better auditability

Reply:
I removed those links to stay within line limits, and for readability. If you feel strongly about it we can add them back in. We could also just have a single comment above `StarkProof` pointing people to Winterfell, and explaining the correspondence.